desktop.css

```diff
@@ -0,0 +1,925 @@
+        font-size: 20px;
+        line-height: 20px;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #FFFFFF;
+    }
+    #item-navigation {
+        width: 255px;
+        height: 40px;
+        display: flex;
+        flex-direction: row;
+        gap: 8px;
+        position: absolute;
+        right: 13%;
+        top: calc(50% - 40px/2);
+    }
+    .navigation-contact {
+        width: 87px;
+    }
+    .navigation-about {
+        width: 69px;
+    }
+    .navigation-contact {
+        width: 83px;
+    }
+    .item-text {
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 600;
+        font-size: 15px;
+        line-height: 20px;
+        display: flex;
+        align-items: center;
+        text-align: center;
+        color: #FFFFFF;
+    }
+    #mail {
+        display: flex;
+        flex-direction: row;
+        align-items: flex-start;
+        padding: 8px;
+        width: 40px;
+        height: 40px;
+        position: absolute;
+        right: 10%;
+        top: calc(50% - 40px/2);
+    }
+    
+    /* Headline section */
+    #headline {
+        position: relative;
+        height: 739px;
+        width: 39%;
+        display: flex;
+        flex-direction: column;
+        gap: 24px;
+        left: 10%;
+        top: 72px;
+        padding: 120px 0px;
+    }
+    .my-name  {
+        width: 100%;
+        height: 160px;
+        font-family: 'Crete Round';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 72px;
+        line-height: 82px;
+        color: #F55800;
+        flex: none;
+        order: 0;
+        flex-grow: 0;
+    }
+    .supporting-text{
+        width: 566px;
+        height: 24px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 500;
+        font-size: 32px;
+        line-height: 44px;
+        display: flex;
+        align-items: center;
+        color: #FFFFFF;
+        flex: none;
+        order: 1;
+        flex-grow: 0;
+    }
+    .description {
+        width: 566px;
+        height: 123px;
+        font-family: 'Inter';
+        font-style: normal;
+        font-weight: 400;
+        font-size: 20px;
+        line-height: 32px;
+        color: #F4F5F7;
+        flex: none;
+        order: 2;
+        flex-grow: 0;
+    }
+    .background-image {
+        display: none;
+    }
+    #desktop-headline-image {
+        position: absolute;
+        height: 100%;
+        width: 43%;
+        top: 72px;
+        right: 0px;
+    }
+    .social-media {
+        display: none;
+    }
+    .desktop-social-media {
+        display: flex;
+        flex-direction: column;
+        gap: 24px;
+        position: absolute;
+        top: 300px;
+        left: 3%;
+    }
+    /* Work section */
+    #recent-works {
+        width: 100%;
+        height: 1705px;
+        background-color: #fff;
+        padding: 0px 10%;
+        position: relative;
+        bottom: 80px;
+    }
```

Comment:
- Hey, please, kindly adjust your design so the work section is well centered in the middle. see the image:

![Screenshot from 2023-01-26 23-04-49](https://user-images.githubusercontent.com/59397159/214961764-77f91dd2-ec28-402d-a5e3-2514f7722bfa.png)